src/content/docs/alerts-applied-intelligence/new-relic-alerts/alert-conditions/quick-start-nrql-condition.mdx

```diff
@@ -0,0 +1,51 @@
+---
+title: Write your first NRQL alert condition
+tags:
+  - Alerts and Applied Intelligence
+  - New Relic Alerts
+  - Alert conditions
+  - NRQL
+metaDescription: How to start writing NRQL alert conditions with a step by step user case.
+---
+
+Use [NRQL queries](/docs/query-your-data/nrql-new-relic-query-language/get-started/introduction-nrql-new-relics-query-language/) to create alert [conditions](/docs/alerts-applied-intelligence/new-relic-alerts/alert-conditions/create-alert-conditions/) with ease.
+
+Your condition only needs two attributes: a [query](/docs/query-your-data/explore-query-data/query-builder/introduction-query-builder/#specify-data) and a [threshold](/docs/alerts-applied-intelligence/new-relic-alerts/alert-conditions/set-thresholds-alert-condition/).
+
+Follow our step-by-step example to write your first alert condition using the NRQL query builder.
+
+<Callout variant="important">
+This is a quick guide for creating NRQL alert conditions. For more information on NRQL alert examples, tips, and troubleshooting, see [Create NRQL alert conditions](/docs/alerts-applied-intelligence/new-relic-alerts/alert-conditions/create-nrql-alert-conditions/). 
+</Callout>
+
+## Step 1: Create your query
+
+Determine what alert condition you want to query before continuing. In our example, we want to create a condition that shows us each host’s CPU utilization. We write out the NRQL query as seen in the image below to track each host individually.
```

Comment:
"Determine what alert condition you want to query before continuing."

If I don't know how to do this, I may be left out in the cold. :)

How can we break this down in a useful way...?

I almost think we could break this up in this:

You can create an alerts condition from any NRQL query. [Info on how to find the query builder.]

[NRQL query screenshot]

[Screenshot caption: Something like: "This query find the average CPU utilization from a system and then breaks it down (facets it) by individual host names."]

Then something like: "Our NRQL query language is a powerful tool that lets you write queries on any entity and its attributes. For more on NRQL, see our [NRQL syntax]."

Just spitballing here :)

In general, I think it's good to avoid using "we". "Our" seems slightly more acceptable as an alternative to "New Relic".